gravitee-apim-portal-webui-next/src/app/log-in/log-in.component.html

```diff
@@ -50,6 +50,9 @@
           Log in
         </button>
       </mat-card-actions>
+      <mat-card-content>
+        <a class="log-in__form__reset" routerLink="/reset-password">Forget password?</a>
```

Comment:
```suggestion
        <a class="log-in__form__reset" routerLink="/reset-password">Forgot password?</a>
```
Also, since you have hard-coded text, you can add `i18n="@@someUniqueIdHere"` to this tag 👍